libgadget/slotsmanager.h

```diff
@@ -124,6 +128,9 @@ struct sph_particle_data
 #define BHP(i) BhP[P[i].PI]
 #define STARP(i) StarP[P[i].PI]
 
+/*Shortcut for the extra data*/
+#define SphP_extra ((struct sph_extra_data*) SlotsManager->info[0].extradata)
```

Comment:
Can we call it something like SphPE?

Reply:
SphP_scratch. I don't like names that differ from another name by just one character.

Reply:
I guess I've been parsing too much pure consonant names...

Reply:
Ah yes, the vowel makes a difference...I've seen several people confuse DESI and DES, for example.